templates/task/cf.yml

```diff
@@ -59,6 +60,7 @@ Resources:
       TaskRoleArn:
         !If [HasTaskRole, !Ref TaskRole, !Ref "AWS::NoValue"]
   DefaultExecutionRole:
+    # An IAM role to grant the Fargate agent to make AWS API calls on your behalf.
```

Comment:
nit: can this wording match that of ExecutionRole yaml (below)?

Reply:
```suggestion
    # An IAM role to control permissions for the containers in your tasks.
```